client/packages/common/src/intl/locales/en/common.json

```diff
@@ -28,6 +28,7 @@
   "button.select-a-color": "Select a colour",
   "button.try-again": "Try again",
   "button.view": "View",
+  "button.delete-lines": "Delete selected lines",
```

Comment:
Moved this from all the individual translation files since it's commonly used in 4 places